src/components/ZclAttributeManager.vue

```diff
@@ -276,6 +276,17 @@ export default {
         return false
       }
     },
+    isDisabledDefault(id, name, selectedClusterId, selectedCluster) {
```

Comment:
Why do you pass "selectedCluster" as function parameter?

Reply:
good catch, will fix, thanks